### ISSUE TITLE
Allow arrow key navigation in Menu

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#5935]
### Expected Behavior
You should be able to navigate in the Menu component with the up and down arrow keys.

### Actual Behavior
You can't navigate with arrow keys, you can only tab through the Menu.

### URL, screen shot, or Codepen exhibiting the issue
See Menu stories on storybook

### Steps to Reproduce
Try using arrow keys to navigate on a Menu
